app-header-layout/app-header-layout.html

```diff
@@ -147,6 +149,32 @@
         this._updateScroller();
       },
 
+      // Resizes app-header-layout to the height of the viewport and translates the content
+      // up to "simulate" disabled document scrolling on iOS.
+      simulateDisabledScroll: function(disabled) {
+        if (disabled) {
+          if (this.hasScrollingRegion) {
+            this.$.contentContainer.style.webkitOverflowScrolling = 'auto';
+          } else {
+            this._savedPageYOffset = window.pageYOffset;
```

Comment:
No, it's a different value. `this._savedPageYOffset` is the _saved_ value of the last scrollTop, and it is only applicable for the document scroller (element scroller can just have `webkitOverflowScrolling` set to `'auto'`, as above).